ksp_plugin/plugin.hpp

```diff
@@ -333,6 +333,11 @@ class Plugin {
   virtual Vector<double, World> VesselNormal(GUID const& vessel_guid) const;
   virtual Vector<double, World> VesselBinormal(GUID const& vessel_guid) const;
 
+  // TODO(egg): UnmanageableVesselTangent, Normal, Binormal.
+
+  virtual Velocity<World> UnmanageableVesselVelocity(
```

Comment:
Please document the new methods.  I have absolutely no idea what they do.  Also a unit test would be good.